sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -1287,6 +1321,23 @@ public enum TextSentiment
         Negative = 2,
         Mixed = 3,
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct WellKnownFhirVersion : System.IEquatable<Azure.AI.TextAnalytics.WellKnownFhirVersion>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public WellKnownFhirVersion(string value) { throw null; }
```

Comment:
Curious question: When .net use wellKnownXXX for the naming

Reply:
I got the name from the thread in this open issue: https://github.com/Azure/azure-sdk-for-net/issues/28811

Reply:
Yeah it is a .NET-specific pattern according to @heaths, see https://github.com/Azure/azure-sdk-for-net/issues/28811#issuecomment-1157098236. In JS, the corresponding pattern is `Known<name>`.